config-connector/solutions/projects/kpt/project-services/README.md

```diff
@@ -0,0 +1,55 @@
+Project Services
+==================================================
+
+# NAME
+
+  project-services
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to enable services on a project.
+  
+## Consumption
+
+  Using kpt:
+  ```
+  BASE=https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+  kpt pkg get $BASE/config-connector/solutions/projects/kpt/project-services project-services
+  ```
+
+## Requirements
+
+  A working cluster with Config Connector installed.
+  The cnrm-system service account must have `roles/serviceusage.serviceUsageAdmin` or `roles/owner`.
```

Comment:
"... must have `roles/serviceusage.serviceUsageAdmin` or `roles/owner` in the desired project"? Otherwise it's confusing where they need to grant the roles.